apps/ios/SimpleXChat/APITypes.swift

```diff
@@ -289,6 +292,7 @@ public enum ChatResponse: Decodable, Error {
     case apiChats(chats: [ChatData])
     case apiChat(chat: ChatData)
     case userSMPServers(smpServers: [String])
+    case sMPTestResult(smpTestFailure: SMPTestFailure?)
```

Comment:
can api be adjusted so that it's all lowercase? or change to "serverTestResult"?
```suggestion
    case smpTestResult(smpTestFailure: SMPTestFailure?)
```

Reply:
Once build compiles